js/hotgraphicPopupView.js

```diff
@@ -39,118 +34,103 @@ define([
-            if (event) event.preventDefault();
-            this.previousHotGraphic();
-        },
-
-        onNextClick: function(event) {
-            if (event) event.preventDefault();
-            this.nextHotGraphic();
-        },
+        onControlClick: function(event) {
+            event.preventDefault();
 
-        previousHotGraphic: function () {
-            var currentIndex = this.model.getActiveItem().get('_index');
-            var canCycleThroughPagination = this.model.get('_canCycleThroughPagination');
-            var itemLength = this.model.get('_items').length;
+            var direction = $(event.currentTarget).hasClass('back') ? 'back' : 'next';
+            var index = this.getNextIndex(direction);
 
-            if (currentIndex === 0 && !canCycleThroughPagination) {
-                return;
-            } else if (currentIndex === 0 && canCycleThroughPagination) {
-                currentIndex = itemLength;
+            if (index !== -1) {
+                this.setItemState(index);
             }
-
-            this.applyItemClasses(currentIndex-1);
         },
-        
-        nextHotGraphic: function () {
-            var currentIndex = this.model.getActiveItem().get('_index');
-            var canCycleThroughPagination = this.model.get('_canCycleThroughPagination');
-            var itemLength = this.model.get('_items').length;
 
-            if (currentIndex === (itemLength-1) && !canCycleThroughPagination) {
-                return;
-            } else if (currentIndex === (itemLength-1) && canCycleThroughPagination) {
-                currentIndex = -1;
+        getNextIndex: function(direction) {
+            var index = this.model.getActiveItem().get('_index');
+            var lastIndex = this.model.get('_items').length - 1;
+
+            switch (direction) {
+                case 'back':
+                    if (index > 0) return --index;
+                    if (this.model.get('_canCycleThroughPagination')) return lastIndex;
+                    break;
+                case 'next':
+                    if (index < lastIndex) return ++index;
+                    if (this.model.get('_canCycleThroughPagination')) return 0;
             }
-
-            this.applyItemClasses(currentIndex+1);
+            return -1;
         },
 
-        applyItemClasses: function(index) {
-            this.setItemState(index);
-
-            this.$('.hotgraphic-item.active').hide().removeClass('active');
-            this.$('.hotgraphic-item').eq(index).show().addClass('active');
-            this.$('.hotgraphic-popup-inner').a11y_on(false);
+        onItemsActiveChange: function(item, _isActive, options) {
+            if (!_isActive) return;
 
+            var index = item.get('_index');
+            this.applyItemClasses(index);
             this.applyNavigationClasses(index);
-            this.handleFocus(true);
             this.updatePageCount();
+            this.handleFocus(!options.initialOpening);
```

Comment:
Could we change the parameter to  `options && !!options.initialOpening` in case `options` aren't defined?